### PR TITLE
Now captcha is working for me

### PR DESCRIPTION
**Pull request by udoersam**
Captcha was always wrong. Now it seems to work correct.

### CODE REVIEW

File: captcha.php [hunk 1]
```diff
@@ -13,11 +13,15 @@
...
-  }
-
-  return $securimage->check($value);
+  }*/
+    if ($securimage->check($value) == true) {
+      return true;
+    } else {
+      return false;
+    }
 };
```
Comment (bjo3rnf) [inline captcha.php:26]: This is actually not different to my code ;) (Except for the commented namespace check) Maybe you set `$namespace` accidentally to some not null value?

Comment (udoersam) [reply captcha.php:26]: Last part is indeed the same.
But I think that this:
if (NULL !== $namespace) {
     $securimage->setNamespace($namespace);
}

is commented out. That caused the problem.

Comment (bjo3rnf) [reply captcha.php:26]: Yes, that's true. I wonder why `$namespace` is not null in your case.

**Comment by bjo3rnf** [comment]
Closed due to lack of feedback. Feel free to reopen in case you need further assistance.

**Pull request closed without merging**